frontstage/views/surveys/help/surveys_help.py

```diff
@@ -0,0 +1,197 @@
+                                   )
+        else:
+            flash('At least one option should be selected.')
+            return redirect(url_for('surveys_bp.get_help_option_select',
+                                    short_name=short_name, business_id=business_id,
+                                    option=option))
+    else:
+        abort(404)
+
+
+@surveys_bp.route('/help/<short_name>/<business_id>/<option>/<sub_option>', methods=['GET'])
+@jwt_authorization(request)
+def get_help_option_sub_option_select(session, short_name, business_id, option, sub_option):
+    """Provides additional options with sub option provided"""
+    if sub_option == 'do-not-have-specific-figures':
+        return render_template('surveys/help/surveys-help-specific-figure-for-response.html',
+                               short_name=short_name, option=option, sub_option=sub_option,
+                               subject='Help answering a survey question',
+                               breadcrumbs=[{"text": "Help completing this survey"}, {}],
+                               business_id=business_id)
+    if sub_option == 'unable-to-return-by-deadline':
+        return render_template('surveys/help/surveys-help-return-data-by-deadline.html',
+                               short_name=short_name, option=option, sub_option=sub_option,
+                               subject='Help answering a survey question',
+                               breadcrumbs=[{"text": "Help completing this survey"}, {}],
+                               business_id=business_id)
+    else:
+        abort(404)
+
+
+@surveys_bp.route('/help/<short_name>/<business_id>/<option>/send-message', methods=['GET'])
+@jwt_authorization(request)
+def get_send_help_message(session, short_name, business_id, option):
+    """Gets the send message page once the option is selected"""
+    if 'errors' in request.args:
+        errors = request.args['errors']
+        flash(errors)
+    if option == 'help-completing-this-survey':
+        breadcrumbs_title = 'Help completing this survey'
+    return render_template('secure-messages/help/secure-message-send-messages-view.html',
+                           short_name=short_name, option=option, form=SecureMessagingForm(),
+                           subject='Help answering a survey question', text_one=breadcrumbs_title,
+                           business_id=business_id
+                           )
+
+
+@surveys_bp.route('/help/<short_name>/<business_id>/<option>/<sub_option>/send-message', methods=['GET'])
+@jwt_authorization(request)
+def get_send_help_message_page(session, short_name, business_id, option, sub_option):
+    """Gets the send message page once the option and sub option is selected"""
+    if 'errors' in request.args:
+        errors = request.args['errors']
+        flash(errors)
+    subject, text_one, text_two = get_subject_and_breadcrumbs_title(sub_option, f'surveys/help/{short_name}/{option}')
```

Comment:
Is it possible to use more descriptive words then text_one and text_two?  What are they for?  It's not clear at a glance?

Reply:
👍